pkg/generate/app/cmd/newapp.go

```diff
@@ -561,7 +552,17 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
 
 func (c *AppConfig) validate() (cmdutil.Environment, cmdutil.Environment, error) {
 	var errs []error
-
+	// Environment File where the parameters are stored
+	if len(c.EnvironmentFile) != 0 {
+		varList, err := app.GetEnvironmentVars(c.EnvironmentFile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Error reading file %v: %v", c.EnvironmentFile, err)
```

Comment:
If you are calling the command with flags that involve more files, this message can be misleading. Use something like "Error reading environment variables from file ...".